newIDE/app/src/AssetStore/AssetCard.js

```diff
@@ -19,6 +22,14 @@ const styles = {
     verticalAlign: 'middle',
     pointerEvents: 'none',
   },
+  previewImagePixelated: {
+    width: '100%',
+    imageRendering: '-moz-crisp-edges',
+    imageRendering: '-webkit-optimize-contrast',
+    imageRendering: '-webkit-crisp-edges',
+    imageRendering: 'pixelated',
```

Comment:
The alternatives are not used, in JS the latest key is overriding the others

Reply:
I've not found how to use multiple values, in an array it didn't work, in a string with or without comma it didn't work.